packages/node_modules/@ciscospark/common/src/one-flight.js

```diff
@@ -47,7 +47,8 @@ export default function oneFlight(...params) {
     let key = prop;
     descriptor.value = wrap(descriptor.value, function oneFlightExecutor(fn, ...args) {
       if (keyFactory) {
-        key = keyFactory(...args);
+        const keyFactoryValue = keyFactory(...args);
+        key = keyFactoryValue || `${key}_${keyFactoryValue}`;
```

Comment:
there's probably no need for the boolean here. In fact, it's still plausible that two methods could produce the same key. 

This'll probably be the safest option:

```js
key = `${key}_${keyFactor(...args)}`;
```